repo2docker/buildpacks/base.py

```diff
@@ -71,6 +71,9 @@
 {% endfor -%}
 {% endif -%}
 
+# So users can move executables here with postBuild, issue #557
```

Comment:
I wouldn't refer to the issue (we will forget which repo etc) and instead include any additional info we do need in the comment here. Personally I think the comment is fine as it is.